### PR TITLE
Add sphinx-tabs package

### PR DESCRIPTION
**Pull request by schmitts**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-sphinx-tabs/package.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    """Create tabbed content in Sphinx documentation when building HTML."""
+
+    homepage = "https://github.com/executablebooks/sphinx-tabs"
+    pypi     = "sphinx-tabs/sphinx-tabs-3.2.0.tar.gz"
+
+    maintainers = ['schmitts']
+
+    version('3.2.0', sha256='33137914ed9b276e6a686d7a337310ee77b1dae316fdcbce60476913a152e0a4')
+
+    depends_on('python@3.6:',       type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-sphinx-tabs/package.py]:
```suggestion
    depends_on('python@3.6:3',       type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-sphinx-tabs/package.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    depends_on('py-setuptools',     type='build')
+    depends_on('py-sphinx@2:',      type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-sphinx-tabs/package.py]:
```suggestion
    depends_on('py-sphinx@2:4',      type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-sphinx-tabs/package.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    depends_on('py-pygments',       type=('build', 'run'))
+    depends_on('py-docutils@0.17:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-sphinx-tabs/package.py]:
```suggestion
    depends_on('py-docutils@0.16.0:0.16', type=('build', 'run'))
```

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-sphinx-tabs/package.py]: You'll also need to add version 0.16 to the py-docutils package

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by alalazo** [comment]
@schmitts

**Comment by schmitts** [comment]
> @schmitts

Forgot about the PR. Will update.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**